form.yml

```diff
@@ -42,5 +42,7 @@ attributes:
     label: "R version"
     help: "This defines the version of R you want to load."
     options:
+      # Define functioning R version overloads using a colon after the module load statement
+      - [ "3.5.0", "intel/16.0.3 R/3.4.2 rstudio/1.1.380_server:R/3.5.0"]
```

Comment:
We don't shell escape the values. I'm not sure if that is a feature or a bug. But we might possibly take advantage of it like so:

```
- [ "3.5.0", "intel/16.0.3 R/3.4.2 rstudio/1.1.380_server; module switch R/3.5.0"]
```

That still seems kind of a hack so this is probably better.